envoy/envoy.yaml

```diff
@@ -0,0 +1,56 @@
+admin:
+  access_log_path: /tmp/admin_access.log
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 9901 }
+
+static_resources:
+  listeners:
+    - name: listener_0
```

Comment:
Can this be more descriptive?

Reply:
Done! Hopefully it's more descriptive haha :sweat_smile: